webpack.config.js

```diff
@@ -19,10 +24,32 @@ var defineEnvPlugin = new webpack.DefinePlugin({
   __ABOUT_MAX_LENGTH__: JSON.stringify(process.env.ABOUT_MAX_LENGTH || null),
   __DEV__: isDev,
   __TEST__: false,
-  __DEV_TOOLS__: (process.env.DEV_TOOLS != null) ? process.env.DEV_TOOLS : (isDev ? true : false)
+  __DEV_TOOLS__: (process.env.DEV_TOOLS !== null) ? process.env.DEV_TOOLS : (isDev ? true : false)
 });
 
-var plugins = [ defineEnvPlugin, new ExtractTextPlugin('style.[contenthash].css') ];
+var plugins = [
+  defineEnvPlugin,
+  new ExtractTextPlugin('style.[contenthash].css'),
+  new CopyWebpackPlugin([
+    {
+      from: 'static',
+      transform: (content, path) => {
+        var code = fs.readFileSync(path, 'utf8');
+        var result = uglifyJS.minify(code);
+        return result.code;
+      }
+    }
+  ]),
+  new HtmlWebpackPlugin({
+    template: 'index.ejs',
+  }),
+  new HtmlWebpackIncludeAssetsPlugin({
+    assets: ['pdfkit.js', 'blob-stream.js'],
+    hash: true,
+    append: true,
```

Comment:
Should the external assets here be included before `bundle.js` (i.e. using `append: false`) to ensure that they're available on the page before the application code attempts to run?